feat: Reorganizing access rights (#543)
apify/apify-docs: sources/platform/collaboration/access_rights.md

---
title: Access rights
description: Manage permissions for your private resources such as Actors, Actor runs, and storages. Allow other users to read, run, modify, or build new versions.
sidebar_position: 12
category: platform
slug: /collaboration/access-rights
---

**Manage permissions for your private resources such as Actors, Actor runs, and storages. Allow other users to read, run, modify, or build new versions.**

---

> You can easily and securely share your own resources - Actors, tasks, key-value stores, datasets, and request queues with other users using a [granular](https://www.google.com/search?client=firefox-b-d&q=define+granular+permissions) permissions system. This enables you to, for example, allow your colleague to run an [Actor](../actors/index.md) or view a [dataset](../storage/dataset.md) but not modify them.

Reviewer: ```suggestion
You can easily and securely share your own resources - Actors, tasks, key-value stores, datasets, and request queues - with other users by using a [granular](https://www.google.com/search?client=firefox-b-d&q=define+granular+permissions) permissions system. This enables you, for example, to let your colleague run an [Actor](../actors/index.md) or view a [dataset](../storage/dataset.md) but not modify it.
```

> To be able to grant access rights to another user, you must have a **username** set in [account settings](https://console.apify.com/account?tab=settings).

You can also grant permission to update an Actor and build a new version.

> [Storages](../storage/index.md) (key-value stores, request queues, and datasets) are sharable the same way with, for example, just **read** permission or a combination of both **read** and **write** permissions.

Reviewer: ```suggestion
[Storages](../storage/index.md) (key-value stores, request queues, and datasets) are sharable in the same way as a **read** permission or a combination of both **read** and **write** permissions.
```

> To share an Actor, task, key-value store, request queue, or dataset, click on the **Actions** button in the top-right corner and select **Share**. You can add a collaborator using their **user ID**, **email**, or **username**. Once you have added the collaborator, you can configure their permissions.

Reviewer: ```suggestion
To share an Actor, task, key-value store, request queue, or dataset, click on the **Actions** button in the top-right corner and select **Share**. You can add a collaborator by using their **user ID**, **email**, or **username**. Once you have added the collaborator, you can configure their permissions.
```
